### ISSUE TITLE
Implement Testing for RegexPro contributions.

### ISSUE THREAD

**Issue by farehiqbal** [hammadsaedi/regex-pro#33]
Contribution is focus on 'making it' but it can break it as well.
There should be some testing scripts that makes sure the contribution/changes do not break the system & the core works just fine.

- Suggest some testing framework.
- write integrations tests.
- Integrate testing system with the issue tracking system. (difficult task)

You get the idea right? This way, before sending a pr, contributor should run the tests, making it easier for the maintainer as well.

(Probably not needed at this stage, but we will need it probably as the project grows).

**Comment by hammadsaedi** [comment]
Yeah, we need testing. Right now, I have no idea about this. I would love to assign the issue who can do the honor. :)

**Comment by bryce-seefieldt** [comment]
Can I take a look at how to implement either MochaJS or Jest testing framework here?
If yes, please let me know if you a preference of which framework to use. Or if you's rather another framework, please advise and I can look at how to implement.

**Comment by hammadsaedi** [comment]
@bryce-seefieldt I've assigned you, You may implement this in any framework you're familiar with. We'll adopt that. :)

**Comment by bryce-seefieldt** [comment]
Thanks for the opportunity . I will get started on this over the weekend.
Just need to close out a few current deadlines before getting into it.
